src/mumble/Log.cpp

```diff
@@ -44,6 +44,7 @@ LogConfig::LogConfig(Settings &st) : ConfigWidget(st) {
 	qtwMessages->setAccessibleName(tr("Log messages"));
 	qsVolume->setAccessibleName(tr("TTS engine volume"));
 	qsbThreshold->setAccessibleName(tr("Length threshold"));
+	qsbMessageThresholdUsers->setAccessibleName(tr("Message threshold users"));
```

Comment:
Is putting this new variable here confusing in regards to the other unaffected variable right above it? (```qsbThreshold``` )

Reply:
Nah, I think it's fine